lib/ansiblelint/__main__.py

```diff
@@ -122,11 +123,15 @@ def report_outcome(matches: List["MatchError"], options) -> int:
         return 0
 
 
-def main() -> int:
+def main(args=None) -> int:
     """Linter CLI entry point."""
+    if args is None:
+        args = sys.argv[1:]
+
     cwd = pathlib.Path.cwd()
 
-    options = cli.get_config(sys.argv[1:])
+    constants.options = get_config(args)
```

Comment:
I don't think you can claim something to be constant if you literally rewrite its value spaghetti-style. Global vars that can be randomly modified from many places in the control flow are something to avoid at any cost.